### ISSUE TITLE
error when using http/get

### ISSUE THREAD

**Issue by zcaudate** [schmee/java-http-clj#1]
I'm getting this error:

```
(./pull '[java-http-clj "0.1.0"])
(require '[java-http-clj.core :as http])
(http/get "http://www.yahoo.com")
```

<img width="525" alt="screen shot 2018-11-06 at 1 10 33 pm" src="https://user-images.githubusercontent.com/1455572/48044144-68ed7700-e1c5-11e8-9f9f-edb75bc13cd9.png">

**Comment by schmee** [comment]
Wow, that is embarresing isn't it? 😅

I've pushed a fix and released version `0.3.0`. In the future debacles like this will be avoided by proper testing. I just finished writing documentation today (https://schmee.github.io/java-http-clj/) and tests are up next. Thanks for reporting!

**Comment by zcaudate** [comment]
cheers. btw. love the library.
